klog.go

```diff
@@ -708,7 +708,7 @@ func (l *loggingT) println(s severity, logr logr.Logger, filter LogFilter, args
 		args = filter.Filter(args)
 	}
 	fmt.Fprintln(buf, args...)
-	l.output(s, logr, buf, file, line, false)
+	l.output(s, logr, buf, 0, file, line, false)
```

Comment:
Nit: I prefer to have a short comment when passing constants to inform the reader what that parameter is, like `0 /* depth */`.